code/modules/cargo/packs/armory.dm

```diff
@@ -260,13 +232,3 @@
 					/obj/item/ammo_box/magazine/wt550m9/wtrubber,
 					/obj/item/ammo_box/magazine/wt550m9/wtrubber)
 	crate_name = "auto rifle ammo crate"
-
-/datum/supply_pack/security/armory/wt550ammo_special
```

Comment:
Same as Tech slugs, also WT-550 AP ammo is weaker then normal we Tx ammo is same damage. No reason to remove these form a balance perspective

Reply:
I literally don't care if it's weak, I care about bypassing techwebs which is a deliberate time gate.

Reply:
Its not a good time gate. In fact they are basiclly if even you dont rush them 9 to 3 mins away at any time